be1-go/channel/election/mod.go

```diff
@@ -25,6 +25,7 @@ import (
 
 const msgID = "msg id"
 const failedToDecodeData = "failed to decode message data: %v"
+const failedToBroadcast = "failed to broadcast message: %v"
```

Comment:
```suggestion
const (
msgID = "msg id"
failedToDecodeData = "failed to decode message data: %v"
failedToBroadcast = "failed to broadcast message: %v"
)
```
What do you think of putting const like that ?